website/src/views/ViewConstants.ts

```diff
@@ -14,9 +14,10 @@ export const singleVariantViewConstants = {
     iconType: 'magnify',
 } as const satisfies ViewConstants;
 
+const nonBreakingHyphen = '\u2011';
```

Comment:
I think your fancy hyphen broke the e2e tests ;) They are probably searching for a normal `-`.